### PR TITLE
chore(cleanup): scale back up annotations-api

### PR DESCRIPTION
**Pull request by bassrock**
## Goal

Scale back up annotations api in preparations for highlights on iOS. 

https://mozilla-hub.atlassian.net/browse/POCKET-9369

### CODE REVIEW

**Review by kschelonka — APPROVED** [review]

**Comment by bassrock** [comment]
Gonna wait until the MySQL 8 upgrade to merge this one.

**Pull request merged**
